ksql-engine/src/main/java/io/confluent/ksql/function/UdfLoader.java

```diff
@@ -213,9 +218,12 @@ private MethodAnnotationMatchProcessor handleUdfAnnotation(final ClassLoader loa
 
       LOGGER.info("Adding UDF name='{}' from path={}", annotation.name(), path);
       final Method method = (Method) executable;
+      final Function<List<Schema>,Schema> schemaProviderFunction =
```

Comment:
a few places in this pr there are no spaces after the `,` - just double check to make sure that we add those (I think checkstyle might check for those)

Reply:
Checkstyle is not complaining about these. Should I add spaces nevertheless?

Reply:
In that case I guess it's up to you! I like the spaces but if we don't have a standard I won't complain